### PR TITLE
Change readme.md header style

### PR DESCRIPTION
**Pull request by dVakulen**
Context: https://github.com/dotnet/orleans/pull/2016#issuecomment-237547121

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,9 @@
-Orleans - Distributed Actor Model
```
Comment (dVakulen) [inline README.md]: This words already appears at the top of the page

File: README.md [hunk 2]
```diff
@@ -1,7 +1,9 @@
...
+<p align="center">
+  <img src="https://github.com/dotnet/orleans/blob/gh-pages/assets/logo_full.png" alt="Orleans logo" width="600px"> 
+</p>
```
Comment (ReubenBond) [inline README.md]: Does the centering of the badges also center the remaining text? I don't any change to the badge code in #2017, but I see a difference in the image you posted.
image

Comment (dVakulen) [inline README.md]: No, I centering of the badge isn't in this PR, wanted to hear opinions about it before of the including into PR.

Comment (dVakulen) [inline README.md]: Also resulting view can be observed by clicking this button: 

chrome_2016-08-04_16-45-12

File: README.md (hunk 1 above)
Comment (jdom) [inline README.md]: what do you mean at the top of the page? I can't find it.

Comment (dVakulen) [inline README.md]: chrome_2016-08-13_01-52-45

**Pull request merged**